modules/xml/xml.h

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2017 Balabit
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 as published
+ * by the Free Software Foundation, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
+ *
+ * As an additional exemption you are allowed to compile & link against the
+ * OpenSSL libraries as published by the OpenSSL project. See the file
+ * COPYING for details.
+ *
+ */
+
+#ifndef XML_H_INCLUDED
+#define XML_H_INCLUDED
+
+#include "parser/parser-expr.h"
+
+typedef struct
+{
+  LogParser super;
+  gchar *prefix;
+  gboolean forward_invalid;
+  GList *exclude_tags;
```

Comment:
Both exclude_tags and exclude patterns are not needed. The exclude_tags set function can compile right away, there is no need to store exclude_tags.